blog-homepage/src/ArticleContent.js

```diff
@@ -0,0 +1,38 @@
+// ArticleContent /////////////////////////////////////////////////////////////
+
+import React from 'react';
+import PropTypes from 'prop-types';
+
+function ArticleContent ({title, description, hasAudioAvailable, memberPreview}) {
+
+  // Determine whether to display the "🔊 Audio available" element.
+  const audioAvailableClasses =
+    hasAudioAvailable ? "audio-is-available" : "audio-is-available hidden";
+
+  // Determine whether to display the "★ Member preview" element.
+  const memberPreviewClasses =
+    memberPreview ? "member-preview" : "member-preview hidden";
```

Comment:
I haven't seen the use of CSS for hiding/showing these elements before. Nice job.